libraries/aws_iam_root_user.rb

```diff
@@ -0,0 +1,28 @@
+# author: Miles Tjandrawidjaja
+class AwsIamRootUser < Inspec.resource(1)
+  name 'aws_iam_root_user'
+  desc 'Verifies settings for AWS root account'
+  example "
+    describe aws_iam_root_user do
+      its('access_key_count') { should eq 0 }
+    end
+  "
+
+  def initialize(conn = AWSConnection.new)
+    @client = conn.iam_client
+  end
+
+  def access_key_count
+    summary_account['AccountAccessKeysPresent']
+  end
+
+  def to_s
```

Comment:
Updated. Output is now:
```
 IAM Root-User
     ✔  access_key_count should eq 0
```

If you are interested in adding the account ID have a look at https://github.com/chef/inspec-aws/pull/54